Bottom: 5b71645d83b5b426b9d6f1e626ed0f59f9199b47
Top:    f6ed14f0d4c074372e06da0fc0ae8e7d128f67f1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-27 00:37:26 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index cec5372..b263b9f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -188,7 +188,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
